components/Layout.tsx

```diff
@@ -138,6 +136,8 @@ const MainNavigation = () => {
   const section = useContext(SectionContext);
   const showMobileNav = useStore((s: any) => s.overlayNavigation === 'docs');
 
+  const { theme } = useTheme();
```

Comment:
Let's make sure light is the default theme when accessing for the first time.

Reply:
Sure